aiocache/base.py

```diff
@@ -418,8 +419,7 @@ async def expire(self, key, ttl, namespace=None, _conn=None):
         :raises: :class:`asyncio.TimeoutError` if it lasts more than self.timeout
         """
         start = time.monotonic()
-        ns = namespace if namespace is not None else self.namespace
-        ns_key = self.build_key(key, namespace=ns)
+        ns_key = self.build_key(key, namespace=namespace)
```

Comment:
I'd be tempted to remove the kwarg from all these, rather than repeating the name.

Reply:
Generally I like to call with keyword args if I think the function signature is likely to change (and of course if some kwargs with defaults are skipped).

But in this case I think it's fine use the positional arguments as you've suggested.  The argument name at least conveys the intent.